tests/integration/golang/aim/app/create_app_test.go

```diff
@@ -58,8 +59,16 @@ func (s *CreateAppTestSuite) Test_Ok() {
 	for _, tt := range tests {
 		s.T().Run(tt.name, func(T *testing.T) {
 			var resp response.App
-			err := s.AIMClient.DoPostRequest("/apps", tt.requestBody, &resp)
```

Comment:
i don't think taking a 1-line client invocation to 10 lines is desirable

Reply:
sorry, it is  reasonable, because otherwise the string length will be more then 120 chars. we have a setting in `golangcilint`:
<img width="1182" alt="Screenshot 2023-09-29 at 16 15 19" src="https://github.com/G-Research/fasttrackml/assets/7852635/de07cf6e-ba91-453b-af9e-6f7b389f58cb">
so we have to satisfy our `linter`. and in general it is much easier to read things when they splited.